crates/ruff/resources/test/fixtures/flake8_pyi/PYI011.pyi

```diff
@@ -101,3 +108,6 @@ def f35(
     x: complex = math.inf  # Error PYI011 Only simple default values allowed for typed arguments
```

Comment:
This comment will be removed.
I am working on another PR to add a method to aggregate comments from `Expression` to keep them.